File: 프로그래머스/lv0/120890. 가까운 수/가까운 수.java
class Solution {
    public int solution(int[] array, int n) {
        int answer = 0;
        int min = 100;
        for(int i:array){
            if(Math.abs(n-i)==min && answer>i){
                answer = i;
            }else if(Math.abs(n-i)<min){
                min = Math.abs(n-i);
                answer = i;
            }
        }
        return answer;
    }
}

File: 프로그래머스/lv0/120890. 가까운 수/가까운 수.js
function solution(array, n) {
    var answer = array.sort((a,b) => {
        if(Math.abs(n-a) !== Math.abs(n-b)) return Math.abs(n-a) - Math.abs(n-b);
        else return a-b;
    });
    return answer[0];
}